config/config.go

```diff
@@ -222,20 +216,41 @@ func NewAgentConfig(yamlPath, netYamlPath string) (*AgentConfig, error) {
 
 	// For Agent 6 we will have a YAML config file to use.
 	if util.PathExists(yamlPath) {
```

Comment:
Should we log an error if this is not the case now that we support only yaml config ?

Reply:
Technically you can start the agent without a config. I'll log a warning though 👍

Reply:
oh but then it could be ran by the network-tracer and not exist nevermind